Core/Graphics/Source/GraphicsImpl.h

```diff
@@ -53,6 +59,7 @@ namespace Babylon
             bool Dirty{};
```

Comment:
I could replace `Initialized` with the thread affinity object, since `m_renderThreadAffinity.is_set()` always has the same value is the current `Initialized` flag. Any thoughts on this?

Reply:
If you add an inline predicate `Initialized()` to `BgfxState` that just checks the affinity, I think that'd be fine. Just leaving it as a bool seems fine too. Having `m_renderThreadAffinity.is_set()` checks in the code without additional context or naming seems like it could be confusing, though.